docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Back up universe data
+headerTitle: Back up universe data
+linkTitle: Back up universe data
+description: Use YugabyteDB Anywhere to back up data.
+aliases:
+  - /preview/back-up-restore-universes/back-up-universe-data/ycql/
+  - /preview/back-up-restore-universes/back-up-universe-data/ysql/
+menu:
+  preview_yugabyte-platform:
+    parent: back-up-restore-universes
+    identifier: back-up-universe-data
+    weight: 20
+type: docs
+---
+
+You can use YugabyteDB Anywhere to back up your YugabyteDB universe data. This includes actions such as deleting and restoring the backup, as well as restoring and copying the database location.
+
+If you are using YBA version 2.16 or later to manage universes with YugabyteDB version 2.16 or later, you can additionally create [incremental backups](#create-incremental-backups) and [configure backup performance parameters](#configure-backup-performance-parameters).
+
+For information on how to schedule backups for a later time or as a recurring task, see [Schedule universe backups](../schedule-data-backups/).
+
+Note that non-transactional backups are not supported.
+
+To view, [restore](../restore-universe-data/), or delete existing backups for your universe, navigate to that universe and select **Backups**.
+
+By default, the list displays all the backups generated for the universe regardless of the time period. You can configure the list to only display the backups created during a specific time period, such as last year, last month, and so on. In addition, you can specify a custom time period.
+
+## Create backups
+
+The **Backups** page allows you to create new backups that start immediately, as follows:
+
+1. Click **Backup now** to open the dialog shown in the following illustration:
+
+    ![Backup](/images/yp/create-backup-new-3.png)
+
+1. Select the API type for the backup.
+
+1. Select the storage configuration you want to use for the backup. For more information, see [Configure backup storage](../configure-backup-storage/).
+
+1. Select the database (YSQL) or keyspace (YCQL) to back up.
+
+1. For YCQL backups, you can choose to back up all tables in the keyspace to which the database belongs or only certain tables. Click **Select a subset of tables** to display the **Select Tables** dialog, where you can select one or more tables to back up. Click **Confirm** when you are done.
+
+1. Specify the period of time during which the backup is to be retained. Note that there's an option to never delete the backup.
+
+1. Optionally, specify the number of threads that should be available for the backup process.
```

Comment:
The UI was not showing me this option . Need to check if this step can be removed. ( specify the number of threads that should be available for the backup process.)